mix.exs

```diff
@@ -20,31 +19,32 @@ defmodule AppPrototype.Mixfile do
   #
   # Type `mix help compile.app` for more information.
   def application do
-    [mod: {AppPrototype, []},
-     applications: [:phoenix, :phoenix_pubsub, :phoenix_html, :cowboy, :logger, :gettext,
-                    :phoenix_ecto, :postgrex]]
+    [mod: {AppPrototype.Application, []},
```

Comment:
Note that application definition moves into its own `Application` - see `application.ex`.